.github/ISSUE_TEMPLATE/feature-request.md

```diff
@@ -0,0 +1,12 @@
+---
+name: Feature Request
+about: Suggest a new feature
+title: ''
```

Comment:
Should we use a tag such as `[FEATURE]` (or something else) to make those as easily identifiable as bug reports?

Reply:
They do get the label `enhancement` which seemed sufficient to me? Since it's quite easy to search and filter by label

Reply:
Sounds good :+1: